src/Microsoft.Diagnostics.NETCore.Client/ReversedServer/ReversedDiagnosticsServer.cs

```diff
@@ -118,6 +106,19 @@ public void Start(int maxConnections)
             _listenTask = ListenAsync(maxConnections, _disposalSource.Token);
         }
 
+        /// <summary>
+        /// Gets endpoint information when a new runtime instance connects to the server.
+        /// </summary>
+        /// <param name="timeout">The amount of time to wait before cancelling the accept operation.</param>
+        /// <returns>An <see cref="IpcEndpointInfo"/> value that contains information about the new runtime instance connection.</returns>
+        public IpcEndpointInfo Accept(TimeSpan timeout)
```

Comment:
That's easily done by the caller using `server.Accept(Timeout.InfiniteTimeSpan)`. I would argue adding `Accept()` falls into the "enabling bad behavior" bucket of things I like to avoid, especially if it's easy for a caller to specify that in order to opt into that behavior if they really want it.